### ISSUE TITLE
Service.currentContext is undefined

### ISSUE THREAD

**Issue by kesslerdev** [moleculerjs/moleculer#676]
## Prerequisites

Please answer the following questions for yourself before submitting an issue. 

- [x] I am running the latest version
- [x] I checked the documentation and found no answer
- [x] I checked to make sure that this issue has not already been filed
- [x] I'm reporting the issue to the correct repository

## Current Behavior
Service.currentContext is undefined, no [documentation update](https://moleculer.services/docs/0.14/services.html#Current-Context-Storage) for this breaking change, no information in [Changelog](https://github.com/moleculerjs/moleculer/blob/next/CHANGELOG.md) or [migration guide](https://github.com/moleculerjs/moleculer/pull/665)

## Failure Information

in moleculer sources it seems commented
https://github.com/moleculerjs/moleculer/blob/master/src/service.js#L421
https://github.com/moleculerjs/moleculer/blob/master/src/service-broker.js#L1549
https://github.com/moleculerjs/moleculer/blob/master/src/service-broker.js#L190

### Steps to Reproduce

Please provide detailed steps for reproducing the issue.

### Reproduce code snippet

```js
const broker = new ServiceBroker({
    logger: console,
    transporter: "NATS"
});

broker.createService({
    name: "test",
    actions: {
        empty(ctx) {
            this.logger.info(this.currentContext)
            return "Hello"
        }
    }
});
```

### Context

Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.

* Moleculer version: 0.14.1
* NodeJS version: v13.2.0
* Operating System: ubuntu

**Comment by AndreMaz** [comment]
Hi @kesslerdev thanks for reporting. Unfortunately we had to remove the `currentContext` due to [huge performance impact caused by `async_hooks`](https://dzone.com/articles/beware-the-performance-cost-of-async-hooks-node-8). I've removed the reference about `currentContext` from the docs to avoid confusing the users and will do the same about the changelog.

**Comment by Jank1310** [comment]
This also breaks the `EventLegacyTraceExporter` because getCurrentContext() has been removed.
```
const ctx = this.broker.getCurrentContext();
		if (ctx)
			this.processExtraMetrics(ctx, payload);
```

**Comment by icebob** [comment]
Ohh, nice catch!

**Comment by icebob** [comment]
Fixed in 09cf905bd75eaa67995026ad886002b4e5863197
